packages/common/client-utils/src/indexBrowser.ts

```diff
@@ -26,3 +26,5 @@ export {
 	TypedEventEmitter,
 	type TypedEventTransform,
 } from "./typedEventEmitter.js";
+
+export { createEmitter, CustomEventEmitter } from "./events/index.js";
```

Comment:
Do we need to export CustomEventEmitter from the package (both here and indexNode.ts)?  Looks like we can get away with only exporting createEmitter, which would be much preferable?

Reply:
Removed `CustomEventEmitter` from the package export: https://github.com/microsoft/FluidFramework/pull/23141/files#diff-929d9bbf8a8aee44bd6d89f563ad253c61656d092e04789bb2ecbdbeb48d50e2